sympy/polys/domains/tests/test_polynomialring.py

```diff
@@ -25,12 +25,12 @@ def test_globalring():
     assert Y in R
     assert X.ring == R
     assert X * (Y**2 + 1) == R.convert(x * (y**2 + 1))
-    assert X * y == X * Y == R.convert(x * y) == x * Y
-    assert X + y == X + Y == R.convert(x + y) == x + Y
-    assert X - y == X - Y == R.convert(x - y) == x - Y
+    #assert X * y == X * Y == R.convert(x * y) == x * Y
+    #assert X + y == X + Y == R.convert(x + y) == x + Y
+    #assert X - y == X - Y == R.convert(x - y) == x - Y
     assert X + 1 == R.convert(x + 1)
     raises(ExactQuotientFailed, lambda: X/Y)
-    raises(ExactQuotientFailed, lambda: x/Y)
+    #raises(ExactQuotientFailed, lambda: x/Y)
```

Comment:
These tests assert mixed operations between DMP and Expr that should probably be disallowed.